instagram-access.sh

```diff
@@ -0,0 +1,6 @@
+    curl -F 'client_id=6171bf0827c2435da76fc87512bb5b1b' \
```

Comment:
I guess this file shouldn't be here?

Reply:
Yeah your right, saved that to the wrong repo.